### ISSUE TITLE
What are the training, evaluation and testing datasets of CodeBERT (the MLM pipeline particularely)?

### ISSUE THREAD

**Issue by Ahmedfir** [microsoft/CodeBERT#271]
Dear Madame or Sir,

Could you please provide us with a list of the projects that have been used for the training (including the evaluation) of CodeBERT? Particularly the CodeBERT-MLM task? 
Any help or guidance is welcome.

Thank you in advance and best regards!

**Comment by guoday** [comment]
Do you mean pre-training code for CodeBERT-MLM model?

**Comment by Ahmedfir** [comment]
Yes, the list of projects that have been used for pre-training the model CodeBERT-MLM. 
For instance, when I load with `Roberta` the model CodeBERT-MLM like so `RobertaForMaskedLM.from_pretrained("microsoft/codebert-base-mlm")`, on which projects this model has been pre-trained? 
This way, we can know which code we can consider as _seen_ or as _unseen_  during the pre-training of CodeBERT-MLM.

**Comment by guoday** [comment]
We don't release pre-training code. We only use train split to pre-train CodeBERT (https://huggingface.co/datasets/code_search_net).

**Comment by Ahmedfir** [comment]
Thank you very much.
